src/NHibernate.Test/Linq/WhereTests.cs

```diff
@@ -432,7 +432,7 @@ public void StringComparisonParamEmitsWarning()
 					AssertStringComparisonWarning(x => x.CustomerId.EndsWith("ANATR", StringComparison.Ordinal), 1);
 					AssertStringComparisonWarning(x => x.CustomerId.IndexOf("ANATR", StringComparison.Ordinal) == 0, 1);
 					AssertStringComparisonWarning(x => x.CustomerId.IndexOf("ANATR", 0, StringComparison.Ordinal) == 0, 1);
-#if NETCOREAPP2_0
+#if NETCOREAPP2_0 || NETCOREAPP2_1
```

Comment:
I have tried with just `#if  NETCOREAPP` instead, but it looks like the async generator does not handle it.

Reply:
For me `NETCOREAPP` is working fine, which .NET Core sdk version are you using? I think that `NETCOREAPP` [was added](https://github.com/dotnet/sdk/pull/2073) in `2.1.3XX`.

Reply:
Sdk 2.0.0, 3.1.417, 5.0.406, 6.0.201. So, I removed that old sdk 2: no more async generation. I have installed a 2.2 SDK: it works fine now, thanks. (But that was not an issue for compiling with VS: strange.)